server/polar/transaction/service/payment.py

```diff
@@ -32,6 +34,24 @@ def __init__(self, charge_id: str, payment_intent_id: str) -> None:
 
 
 class PaymentTransactionService(BaseTransactionService):
+    async def get_by_charge_id(
```

Comment:
Duplicate of `_get_by_charge_id`. You can make it non-private if you need it elsewhere.